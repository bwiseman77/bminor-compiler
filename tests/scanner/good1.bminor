/* *** ** help! 
 *
 *
 *
 */

// this should all be ok!

/* hdhjjahjkdhajkds
jksdj
kjadlkjfd */
