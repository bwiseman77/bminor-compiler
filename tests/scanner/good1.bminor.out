-------------------------------------------
           token type  text
-------------------------------------------
  MULTI_LINE_COMMENT  
 SINGLE_LINE_COMMENT  // this should all be ok!
  MULTI_LINE_COMMENT  
